FAQ.md

```diff
@@ -57,6 +57,12 @@ Gated models simply require login and in some case acceptance of model provider
 
 NVIDIA Corporation officially contributes to the garak open-source project and will continue to do so in the long term. Garak will continue to be licensed with Apache 2.0. Get in touch if you'd like to talk more about this.
 
+## Can an LLM have vulnerabilities?
+
+The things garak probes for are generally not like traditional cybersec vulnerabilities. LLM model parameters don't and can't have vulnerabilities themselves; it's just data -- just like a theoretical Turing machine isn't vulnerable, it just runs programs.* What most of the probes in garak check for are whether or not a model can be made to behave unexpectedly at inference time, by breaking its alignment or output policy, using exploits. The DHS calls some of these behaviours "weaknesses"; see e.g. [CWE-1426](https://cwe.mitre.org/data/definitions/1426.html) for prompt injection. 
```

Comment:
I think I'd pull the example of the theoretical Turing machine: https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-32471